platform/cli/templates/mode/src/toolbarButtons.js

```diff
@@ -0,0 +1,608 @@
+// TODO: torn, can either bake this here; or have to create a whole new button type
+// Only ways that you can pass in a custom React component for render :l
+import {
```

Comment:
I'm not sure I follow what you say here, are you saying in the cli tool I copy the file from the longitudinal mode? since modes depending on other modes are just not good